pycheribuild/projects/cross/openssl.py

```diff
@@ -34,15 +34,19 @@
 
 
 class BuildOpenSSL(CrossCompileProject):
-    # Just add the FETT target below for now.
-    do_not_add_to_targets = True
     build_in_source_dir = True
+    path_in_rootfs = "/usr/local"
```

Comment:
Could benefit from a comment why the default doesn't work (namely that nginx's home-grown configure scripts suck...)